tests/orders/tests-edd-payment.php

```diff
@@ -911,6 +911,26 @@ public function test_get_tax_rate_returns_decimal() {
 		$this->assertEquals( 0.2, $payment->tax_rate );
 	}
 
+	public function test_receipt_show_download_files() {
```

Comment:
```suggestion
	public function test_receipt_show_download_files_converts_array_to_order_item() {
```

Just a little more descriptive. :smile: